src/lib/mina_lib/mina_lib.ml

```diff
@@ -2182,3 +2183,540 @@ let runtime_config { config = { precomputed_values; _ }; _ } =
   Genesis_ledger_helper.runtime_config_of_precomputed_values precomputed_values
 
 let verifier { processes = { verifier; _ }; _ } = verifier
+
+let%test_module "Epoch ledger sync tests" =
```

Comment:
The `mina_lib.ml` file is already so huge, is it possible to move theses tests into a separate file? For instance, having a test-only sub-library under mina_lib would make sense to me (similar to what was done for `lib/transition_frontier/tests` (although even a separate `lib/mina_lib/tests.ml` file would be enough here).